packages/@o3r/pipeline/schematics/ng-add/index.ts

```diff
@@ -57,7 +57,12 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
           tree.create(npmrcPath, npmRegistry);
         } else {
           const npmrcContent = tree.readText(npmrcPath);
-          tree.overwrite(npmrcPath, npmrcContent.concat(`\n${npmRegistry}`));
+          const registryPattern = /^registry=.*$/m;
+          const newRegistryLine = `registry=${npmRegistry}`;
+          const newContent = registryPattern.test(npmrcContent)
```

Comment:
Why do you want to enforce your registry if already a default one is specified?

Reply:
Only when a custom one is passed in the schematic's options. 
Today when we generate an npm otter monorepo it comes with `registry=http://registry.npmjs.org/`